### ISSUE TITLE
Cascading Notifications

### ISSUE THREAD

**Issue by Eth3rnit3** [simukappu/activity_notification#127]
Is there a current way to proceed to the sending of notifications of the different notifications in a sequenced way?

For example a notification is created, if it is not read 10min later it is sent on slack then 10min later by mail.

**Comment by simukappu** [comment]
No, the gem does not provide a function to send delayed notifications in a sequenced way. I think you can implement this using your own *ActiveJob* job.
If you could provide any PR as the gem enhancement, I would appreciate it. Thank you!

**Comment by Eth3rnit3** [comment]
Thank you for your response!

I'm probably going to have to implement it, so if that's the case, I'll make sure it's easily applicable to all notifiers. I'll let you know when I have something clean and functional.

**Comment by simukappu** [comment]
Published as v2.5.0
